src/server/worldserver/worldserver.conf.dist

```diff
@@ -4119,8 +4119,8 @@ AntiCheats.IgnoreControlMovement.Enabled = 1
 ###################################################################################################
 # Logs for Anticheat
 
-Appender.Anticheat=2,3,15,Anticheat.log
-Logger.anticheat=3,Console Anticheat
+#Appender.Anticheat=2,3,15,Anticheat.log
```

Comment:
Now config with additional file with logs commited. All info in auth-database in anticheat_logs.

Reply:
are old logs ever cleared ?

Reply:
> are old logs ever cleared ?

what you mean? .log files? as you wish. If about data in anticheat_logs - nope.